web/src/domain/constants.ts

```diff
@@ -265,6 +265,11 @@ export const DownloadClientTypeOptions: RadioFieldsetOption[] = [
     label: "Readarr",
     description: "Send to Readarr and let it decide",
     value: "READARR"
+  },
+  {
+    label: "Porla",
+    description: "Add torrents directly to Porla",
+    value: "PORLA"
```

Comment:
Petition to move this to just below Transmission, so its hanging out with the download client gang and not sitting alone under the arr suite.

<img width="684" alt="image" src="https://user-images.githubusercontent.com/18177310/215262906-761ea341-9981-43cf-a31b-5eb4c99efd61.png">

Reply:
makes sense to me

Reply:
agree too.